examples/MyModule/recording_backend_soundclick.cpp

```diff
@@ -0,0 +1,133 @@
+/*
+ *  recording_backend_soundclick.cpp
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+// Includes from libnestutil
+#include "compose.hpp"
+
+// Includes from nestkernel
+#include "recording_device.h"
+#include "vp_manager_impl.h"
+
+// Includes from sli
+#include "dictutils.h"
+
+// Own includes
+#include "recording_backend_soundclick.h"
+
+
+// Constructor
+nest::RecordingBackendSoundClick::RecordingBackendSoundClick()
+{
+  // Load the raw sound data into the SFML sound buffer.
+  sound_buffer_.loadFromMemory(
+    sound_click_16bit_44_1khz_wav, sizeof( sound_click_16bit_44_1khz_wav ) );
+  sound_.setBuffer( sound_buffer_ );
+  LOG( M_INFO,
+    "Recording Backend",
+    ( "recording backend >SoundClick< successfully initialized" ) );
+}
```

Comment:
Please move this code to the `initialize()` function. I've fixed the problem of that function not being run in 699c0698e.

Reply:
Moved the log message to initialize() but left the SMFL object instantiation in the constructor, according to RAII.